src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/DatePicker/DatePickerSample.xaml

```diff
@@ -0,0 +1,28 @@
+﻿<Page x:Class="SamplesApp.Samples.DatePicker.DatePickerSample"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:local="using:ControlTests"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+      xmlns:xamarin="http://nventive.com/xamarin"
+      xmlns:ios="http://nventive.com/ios"
+      xmlns:android="http://nventive.com/android"
+      xmlns:native_ios="using:UIKit"
+      xmlns:native_android="using:Android.Widget"
+      xmlns:controls="using:Uno.UI.Samples.Controls"
+      mc:Ignorable="xamarin android ios native_ios native_android">
+
+	<controls:SampleControl SampleDescription="Sample for Date Picker">
+		<controls:SampleControl.SampleContent>
+			<DataTemplate>
+				<Grid Background="White">
+					<DatePicker Grid.Row="1"
+								Margin="15,10"
+								HorizontalAlignment="Center"
+								VerticalAlignment="Top"
+								FlyoutPlacement="Full">
```

Comment:
sample doesn't have to specify FlyoutPlacement to work properly if I understand correctly? it's in the default style?
I would remove it from here